utils/rpms/daos.spec

```diff
@@ -346,11 +347,18 @@ getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r -g daos_serv
 %post server
 /sbin/ldconfig
 %systemd_post %{server_svc_name}
+set -o noclobber
+cat << EOF > %{sysctl_script_name}
+net.core.rmem_max=104857 +
+net.core.rmem_default=104857
+EOF
```

Comment:
Hrm.  This seems better as a _config_ file in the RPM rather than being done in a `%post` scriptlet.  That way it's contents would be preserved and treated as a configuration during upgrades, making `.rpm{save|new}` files when the contents change, etc.

This also alleviates the need/desire for `set -o noclobber` and having an install/upgrade's `%post` erroring out if the file does exist.